bids/tests/utils.py

```diff
@@ -1,7 +1,19 @@
-''' Test-related utilities '''
+""" Test-related utilities """
 
-from os.path import join, dirname, abspath
+from pathlib import Path
 
 
-def get_test_data_path():
-    return join(dirname(abspath(__file__)), 'data')
+def get_test_data_path(return_type="str"):
+    """
+    :param return_type: Specify the type of object returned. Can be 'str'
+                        (default, for backward-compatibility) or 'Path' for pathlib.Path type.
+    :return: The path for testing data.
+    """
```

Comment:
Please use a consistent docstring format. Again, I think it's numpydoc we've been using.